tests/unit/components/app-popover/AppPopover.spec.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import AppPopover from '~/components/app-popover/AppPopover'
+import { vi } from 'vitest'
+
+describe('AppPopover component', () => {
+  const props = {
+    initialItems: <div data-testid='initial'>Initial items</div>,
+    showMoreElem: <button>Show more</button>,
+    initialItemsWrapperStyle: {
+      backgroundColor: 'red'
+    }
+  }
+  const childrenComponent = <div>Children</div>
+
+  vi.mock('@mui/material/Popover', () => {
+    return {
+      default: function MockOpenButton(props) {
```

Comment:
maybe we should destructure all `props` fields here? Like
```suggestion
      default: function MockOpenButton({onClose, open, ...}) {
```